fiasco/base.py

```diff
@@ -22,8 +23,11 @@ def __init__(self, ion_name, hdf5_dbase_root=None, **kwargs):
         self._element, ion = ion_name.split()
         if '+' in ion:
             ion = f"{int(ion.strip('+')) + 1}"
+        if roman.is_roman_numeral(ion):
+            ion = roman.from_roman(ion)
         self.ionization_stage = int(ion)
         self.charge_state = self.ionization_stage - 1
+        self.roman_symbol = roman.to_roman(int(ion))
```

Comment:
Could you make this a property instead? And maybe add a `roman_name` as well? e.g.

```python
@property
def roman_numeral(self):
    return roman.to_roman(self.ionization_stage)

@property
def roman_name(self):
    return f'{self.atomic_symbol} {self.roman_numeral}'
```

Reply:
I've used `numeral` here instead of `symbol` since `symbol` sort of clashes with `atomic_symbol`